ansible_ai_connect/ai/api/model_pipelines/http/pipelines.py

```diff
@@ -309,3 +329,146 @@ async def async_invoke(self, params: StreamingChatBotParameters) -> AsyncGenerat
                     }
                     yield json.dumps(error).encode("utf-8")
                     return
+
+
+@Register(api_type="http")
```

Comment:
I was thinking @TamiTakamiya 

Use of llama-stack is really a different type of _provider_ and not a new _feature_.

Therefore, the should be a new `t_model_mesh_api_type` [here](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/main/settings/types.py#L22-L32).

You could copy/paste the entire `http` folder, renaming it something like `llamastack` and move this pipeline implementation to it. L#334 would be `@register(api_type="llamastack")` etc. The [`provider`](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/main/settings/legacy.py#L187) in `ANSIBLE_AI_MODEL_MESH_CONFIG` for ChatBot would be `llamastack`.

Does this make sense to you?

Reply:
Yes, it does and it looks cleaner. My original intension was just to have the llama stack pipeline with keeping the existing streaming chat pipeline as it is.